src/cBiff12Writer.cls

```diff
@@ -105,10 +109,11 @@ End Property
 ' Methods
 '=========================================================================
 
-Public Function Init( _
+Public Function init( _
```

Comment:
Can we keep `Init` function name casing?

Reply:
Yes, Case sesitivity is not made by myself. I think vb6 IDE make the changes, maybe the german version i use.